lib/realtime_signs_web/controllers/monitoring_controller.ex

```diff
@@ -39,4 +40,17 @@ defmodule RealtimeSignsWeb.MonitoringController do
 
     send_resp(conn, 200, "")
   end
+
+  def update_active_headend(conn, %{"ip" => active_ip} = _params) do
+    case Headend.update_active_headend_ip(active_ip) do
+      {:ok, _} ->
+        send_resp(conn, 200, "ok")
+
+      {:bad_request, message} ->
+        send_resp(conn, 400, message)
+
+      {:error, _} ->
+        send_resp(conn, 500, "Internal server error")
```

Comment:
It probably doesn't matter on way or another, but do we know if ARINC is planning on doing anything with the returned status code?

Reply:
Not certain yet. I added the status codes to hopefully make it easier if ARINC wants to do something with them, and to provide a basic level of feedback for cases where we'd need to remote into a headend server and make an http request ourselves